deku-p/src/core/bin/node/deku_node.ml

```diff
@@ -129,7 +127,6 @@ let main params style_renderer log_level =
   (* The VM must be started before the node because this call is blocking  *)
   Logs.info (fun m ->
```

Comment:
This log message is misleading without an actual process/ipc being started, I think it could be removed alongside with the `External_vm_client.start_vm_ipc` call.